### DISCUSSION TITLE
windows version: Error getting git version: exec: "git": executable file not found in %PATH%

### DISCUSSION THREAD

**Discussion by belinaj** [git-lfs/git-lfs#4653] [Q&A] [answered]
Hello, 

installed GIT LFS Windows version. 
"C:\Program Files\Git LFS" is in PC PATH.

However when I try to initialize it per instructions:

Download the windows installer 
Run the windows installer
Start a command prompt/or git for windows prompt and run:  git lfs install

Does not work:

```
C:\Program Files\Git LFS>dir
 Volume in drive C is System
 Volume Serial Number is 9CA2-5C7F

 Directory of C:\Program Files\Git LFS

27.09.2021.  13:39    <DIR>          .
27.09.2021.  13:39    <DIR>          ..
26.03.2021.  15:00        10.126.968 git-lfs.exe
27.09.2021.  13:39             6.119 unins000.dat
27.09.2021.  13:39         3.067.965 unins000.exe
               3 File(s)     13.201.052 bytes
               2 Dir(s)  300.269.838.336 bytes free

C:\Program Files\Git LFS>git lfs install
'git' is not recognized as an internal or external command,
operable program or batch file.

C:\Program Files\Git LFS>
```
 
I have tried also:

```
C:\Program Files\Git LFS>git-lfs.exe install
**Error getting git version: exec: "git": executable file not found in %PATH%**

C:\Program Files\Git LFS>
```

What am I missing?

TYWM in advance.

**Comment by bk2204** [comment, marked as answer]
Hey,

Have you installed Git for Windows?  That's a necessary requirement for Git LFS to work.  If you have, you'll need to make sure it's in your `PATH` as well.

Note that Git for Windows usually includes a version of Git LFS as part of it, so you may wish to just use that version instead of installing a separate one.

**Reply by belinaj** [reply]
Thanks! Maybe to add this to howtos?
